openshift/template.yaml

```diff
@@ -25,7 +25,6 @@ parameters:
 - name: OPENSHIFT_RELEASE_SYNCER_INTERVAL
   value: "30m"
   required: false
-  required: false
```

Comment:
I think the `OPENSHIFT_RELEASE_SYNCER_INTERVAL` parameter is declared twice. A bit of a mess but shouldn't we had removed one of those?